apps/methodologies/bold-carbon-organic/rule-processors/credit/nft-metadata-selection/src/lib/nft-metadata-selection.helpers.spec.ts

```diff
@@ -237,7 +237,7 @@ describe('Helpers', () => {
 
       const validation = validate<MassMetadata>(result);
 
-      expect(validation.errors).toEqual([]);
+      expect(validation.success).toBe(true);
```

Comment:
In the previous approach, it was more dev friendly as we could view the errors when the expect fails. In Palantir, we kept the same behaviour as before by creating a custom matcher. See this [dev-tip](https://carrot-fndn.slack.com/archives/C027VFT6T6J/p1738761786672689) [this commit](https://github.com/carrot-foundation/palantir/pull/2764/commits/cb36bee1e0bdb94c949c8b926ca3bc516c471887) and [this PR](https://github.com/carrot-foundation/palantir/pull/2764) as reference